src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -360,4 +360,25 @@ public static String sanitizeStringForXPath(String text) {
         }
         return "concat(" + result.toString() + "'')";
     }
+
+    /**
+     * Returns true if the {@code string} has evidence of having been sanitized.
+     * A string is considered sanitized if it does not contain any of the chars '<', '>', '/', '\"', '\'',
+     * and contains at least one of their sanitized equivalents or the sanitized equivalent of '&'.
+     */
+    public static boolean isSanitizedHtml(String string) {
+        return string != null
+                && !StringHelper.isTextContainingAny(string, "<", ">", "\"", "/", "\'")
```

Comment:
Maybe my point wasn't clear, but this should go into the documentation comment above the method. There is no point repeating that
```
* A string is considered sanitized if it does not contain any of the chars '<', '>', '/', '\"', '\'',
* and contains at least one of their sanitized equivalents or the sanitized equivalent of '&'.
```
when that information is available in the code itself. Instead, you should explain _why_.